### ISSUE TITLE
ocp_installer.sh should use private key

### ISSUE THREAD

**Issue by longdafeng** [oceanbase/oceanbase#744]
in the configuration file: 

```
SSH_USER=root    # if not root, make sure remote user can use sudo without password    admin   ALL=(ALL)   NOPASSWD:ALL
SSH_PORT=22
SSH_AUTH=pubkey     # can be password or pubkey
SSH_PASSWORD=''     # password for passowrd auth, when use pubkey auth, is passphrase
SSH_KEY_FILE='/root/.ssh/id_rsa.pub'   # pubkey auth
```

SSH_KEY_FILE='/root/.ssh/id_rsa.pub'   # pubkey auth,  it should be private key, not public key
it should be SSH_KEY_FILE='/root/.ssh/id_rsa'

**Comment by chris-sun-star** [comment]
yes, the default value is the public key, should change to private key

**Comment by chris-sun-star** [comment]
fix default value in version 3.1.1-ce-bp1
